README.md

```diff
@@ -21,6 +23,9 @@ Colo Loco will find your colocated native files, automatically link them up to t
 
 ### Watch Jamon show how to use React Native Colo Loco!
 
+[Watch the tutorial on YouTube](https://www.youtube.com/watch?v=gfORZXq4ZgE)
```

Comment:
Move this line to a new section, `# Contributing to React Native Colo Loco` 

Don't call it a Tutorial -- it should be a demo of how to contribute.